tools/info/main.cpp

```diff
@@ -378,6 +442,8 @@ void GatherVulkanStats(const std::string& input_filename)
         vulkan_decoder.AddConsumer(&vulkan_stats_consumer);
         file_processor.AddDecoder(&vulkan_decoder);
 
+        file_processor.SetAnnotationProcessor(&vulkan_stats_consumer);
```

Comment:
@andrew-lunarg , please merge #1177 ; that PR already contains this fix

Reply:
Done